ui/src/backoffice/authors/containers/__tests__/AuthorDetailPageContainer.test.tsx

```diff
@@ -50,6 +53,14 @@ describe('AuthorDetailPageContainer', () => {
     expect(asFragment()).toMatchSnapshot();
   });
 
+  it('should display ORCID link only once', () => {
+    renderComponent();
+
+    const link = screen.getByRole('link', {name: '0000-0002-6357-9297'});
+    expect(link).toBeInTheDocument();
+    expect(link).toHaveAttribute('href', expect.stringContaining('/orcid.org/0000-0002-6357-9297'));
```

Comment:
if you have the whole url already you  do not need to do stringContaining. You can check against the full href. 

E.g 
```
expect(link).toHaveAttribute('href', 'https://www.springer.com/journal/526');
```